### PR TITLE
/npc mes ""; to mes("");

### PR DESCRIPTION
**Pull request by Habilya**
[//]: # (**********************************)
[//]: # (** Fill in the following fields **)
[//]: # (**********************************)

[//]: # (Note: Lines beginning with syntax such as this one, are comments and will not be visible in your report!)

### Pull Request Prelude

[//]: # (Thank you for working on improving Hercules!)

[//]: # (Please complete these steps and check the following boxes by putting an `x` inside the brackets _before_ filing your Pull Request.)

- [x] I have followed [proper Hercules code styling][code].
- [x] I have read and understood the [contribution guidelines][cont] before making this PR.
- [x] I am aware that this PR will be closed if the above-mentioned criteria are not fulfilled.

### Changes Proposed

[//]: # (Describe at length, the changes that this pull request makes.)

**Affected Branches:** 
master
[//]: # (Master? Slave?)

**Issues addressed:**
https://github.com/HerculesWS/Hercules/issues/2233
[//]: # (Issue Tracker Number if any.)

### Known Issues and TODO List

[//]: # (Insert checklist here)
[//]: # (Syntax: - [ ] Checkbox)

[//]: # (**NOTE** Enable the setting "[√] Allow edits from maintainers." when creating your pull request if you have not already enabled it.)

[cont]: https://github.com/HerculesWS/Hercules/blob/master/CONTRIBUTING.md
[code]: https://github.com/HerculesWS/Hercules/wiki/Coding-Style

### CODE REVIEW

**Comment by HerculesWSAPI** [comment]
This change is [Reviewable](https://reviewable.io/reviews/herculesws/hercules/2257)

**Comment by dastgirp** [comment]
Please fix the errors: https://travis-ci.org/HerculesWS/Hercules/jobs/437303667

**Comment by 4144** [comment]
please squash fixes commits into other commits. because fix commits for non merged commits is useless

File: npc/re/cities/mora.txt [hunk 1]
```diff
@@ -525,9 +525,9 @@ mora,140,186,3	script	Raffle Philosopher	4_M_RAFLE_OLD,{
 		mes("- John Dewey -");
 		break;
 	case 8:
-		mes("We sow our thoughts, and we reap our actions");
-		mes("we sow our actions, and we reap our habits");
-		mes("we sow our habits, and we reap our characters");
+		mes("We sow our thoughts, and we reap our actions;");
```
Comment (Emistry) [inline npc/re/cities/mora.txt]: extra `;`

File: npc/re/jobs/2e/kagerou_oboro.txt [hunk 2]
```diff
@@ -1778,7 +1778,7 @@ OnTouch:
 	mes("You get to pass if you get 4 or above but nobody knows what will come from a dice roll.");
 	next;
 	mes("Rule 2");
-	mes("If you get 1, 2, or 3 you move right to the arrival; if you get 4, 5, or 6 you get to choose to move to block 29.");
+	mes("If you get 1, 2, or 3 you move right to the arrival point; if you get 4, 5, or 6 you get to choose to move to block 29.");
```
Comment (Emistry) [inline npc/re/jobs/2e/kagerou_oboro.txt]: extra `;`

**Comment by dastgirp** [comment]
As per request, closing it

**Pull request closed without merging**